repos/system_upgrade/el7toel8/actors/targetuserspacecreator/actor.py

```diff
@@ -25,5 +27,6 @@ class TargetUserspaceCreator(Actor):
     tags = (IPUWorkflowTag, TargetTransactionFactsPhaseTag)
 
     def process(self):
-        if version.is_supported_version() and next(self.consume(RepositoriesMap), None):
+        if ((version.is_supported_version() or os.getenv('LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE'))
```

Comment:
use the configuration api to get the value, instead of os.getenv.

Reply:
Alright.